plugins/lookup/passwordstore.py

```diff
@@ -138,6 +138,7 @@
 import subprocess
 import time
 import yaml
+import gpg
```

Comment:
Is this https://pypi.org/project/gpg/ (from 2018)?

Also please note that plugins are not allowed to import third-party modules this way, you need to use a guarded import (see https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/import.html for details).

Reply:
Yes, I think it is

Reply:
It is yes (source: I wrote this module for the project @sigio is working on)